audbackend/core/api.py

```diff
@@ -62,15 +63,16 @@ def create(
         backend object
 
     Raises:
+        BackendError: if an error is raised on the backend
```

Comment:
This sounds a little bit too generic to me.
Maybe change to
```suggestion
        BackendError: if an error is raised on the backend
            when trying to access or create the repository
```
Or are there other interactions which could lead to this error?

Reply:
I guess so, e.g. a connection error. So maybe we leave it as is but add more explanation what might get wrong in the description as suggested in https://github.com/audeering/audbackend/pull/91#issuecomment-1514293952

Reply:
OK, makes sense.